report_templates/index.Rmd.in

```diff
@@ -734,14 +735,12 @@ cat('### Overview of Hyper- and Hypo-Methylated CpGs Over the Genome\n')
 
 if(length(GRanges.diffmeth.hypo)>1 & length(GRanges.diffmeth.hyper)>1){
 
-require(GenomeInfoDb)
```

Comment:
Can we drop GenomeInfoDb from the dependencies then?

Reply:
I'll remove ` library("GenomeInfoDb")` from the index.Rmd file on the next commit and re-test